libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1522,6 +1523,13 @@ bool WaveTrack::RateConsistencyCheck() const
       });
 }
 
+void WaveTrack::PushBack(WaveClipHolder clip)
```

Comment:
I’d like a different name for this function, like `InsertClip`

I may want to strengthen the internal invariant, so that clips are in fact maintained in sorted sequence.

The function name should not suggest something about the position of the clip in the list.

Reply:
Good idea.